xclim/indicators/atmos/_precip.py

```diff
@@ -308,3 +310,23 @@ class HrPrecip(Hourly):
         indices.liquid_precip_ratio, suggested={"tas": _empty}, prsn=None
     ),
 )
+
+
+rolling_drydays_events = Precip(
+    identifier="rolling_drydays_events",
+    description="The number of periods of minimum {window} days during which the accumulated precipitation is under"
+    "{thresh}, for each resampling period.",
+    units="",
+    cell_methods="",
+    compute=indices.rolling_drydays_events,
+)
+
+
+rolling_drydays_count = Precip(
+    identifier="rolling_drydays_count",
+    description="The number of days in periods of minimum {window} days during which the accumulated precipitation "
```

Comment:
Same comments here regarding `{freq}` and `{window}`  as for 'events' calculations

Reply:
Are you suggesting to replace "for each resampling period" by {freq}?

Reply:
> Are you suggesting to replace "for each resampling period" by {freq}?

nope see above